spring/boot2-webflux-autoconfigure/src/main/java/com/linecorp/armeria/spring/web/reactive/ArmeriaClientHttpRequest.java

```diff
@@ -141,7 +141,7 @@ public DataBufferFactory bufferFactory() {
             assert request == null : request;
             request = supplier.get();
             future.complete(client.execute(request));
-            return Mono.fromFuture(request.whenComplete());
+            return Mono.empty();
```

Comment:
Don't we need to complete `Mono` lazily? If then, how about?
```java
Mono.fromFuture(future)
    .onErrorResume(unused -> Mono.empty())
```

Reply:
If so, we cannot get the response until the request is complete which means that we cannot use bidi streaming request.
So the `Mono` should not depend on whether the request is complete or not. 😉

Reply:
Ah... I see. Let's leave as it is.